src/main/resources/db/changelog/changes/sharedlinks/v1.0-sharedlinks-changelog.sql

```diff
@@ -0,0 +1,27 @@
+--liquibase formatted sql
+
+--changeset przemyslawren:create-links-table
+CREATE TABLE IF NOT EXISTS shared_links.links (
+    id              UUID PRIMARY KEY,
+    user_id         UUID,
+    routine_id      UUID,
+    link            VARCHAR(256) NOT NULL,
+    expires_at      TIMESTAMP NOT NULL,
+    is_active       BOOLEAN,
+    created_at      TIMESTAMP NOT NULL DEFAULT NOW(),
```

Comment:
we probably should be consistent about using regular TIMESTAMP or the ones with time zone. If we choose the former then lets also update existing changelogs